### PR TITLE
zh: add content_zh/docs/tasks/telemetry/access-log/index.md

### PR DESCRIPTION
**Pull request by haiker2011**
fix: servicemesher/istio-official-translation#830

### CODE REVIEW

File: content_zh/docs/tasks/telemetry/logs/access-log/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --namespace=istio-system -x templates/configmap.yaml --set global.proxy.accessLogFile="/dev/stdout" | kubectl replace -f -
+configmap "istio" replaced
+{{< /text >}}
+
+您也可以通过设置 `accessLogEncoding` 来在 JSON 和 TEXT 两种格式之间切换。
+
+您也许希望通过 `accessLogFormat` 来自定义
+[format](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#format-rules) 访问日志。
```
Comment (fleeto) [inline content_zh/docs/tasks/telemetry/logs/access-log/index.md]: 是这个意思么？

您也许希望通过 `accessLogFormat` 来自定义
访问日志的[格式](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#format-rules) 。

Comment (haiker2011) [reply content_zh/docs/tasks/telemetry/logs/access-log/index.md]: 嗯嗯，是的，我修改一下。

**Comment by haiker2011** [comment]
@fleeto 已修改 review 意见

**Comment by fleeto** [comment]
`[trans-bot:merged]`

Thank you @haiker2011, the [related task issue](https://github.com/servicemesher/istio-official-translation/issues/830) had been updated.

**Pull request merged**